packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -181,17 +196,18 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
 
         // These are set in ascending order of their UpdateTypes.
         _setBatcherHash(_batcherHash);
-        _setGasConfig({ _overhead: _overhead, _scalar: _scalar });
+        _setGasConfig({ _overhead: _gasConfig.overhead, _scalar: _gasConfig.scalar });
```

Comment:
Thoughts on not unwrapping the type here? Did you not do that to preserve the abi?

Reply:
Good call out.